recipes/noiseflow/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+python:
+  - 3.9
+  - 3.10
+  - 3.11
```

Comment:
Usually you would not have to make this file.

Reply:
Yes, now I don't know how to fix the Python versions for the package, so I add those for a try